browser/components/molecules/Timeline/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component, PropTypes } from 'react'
+import moment from 'moment'
+import getPrrrsPerWeek from '../../../actions/getPrrrsPerWeek'
+import Button from '../../atoms/Button'
+import './index.sass'
+import Link from '../../atoms/Link'
+
+export default class Timeline extends Component {
+  static propTypes = {
+    week: PropTypes.instanceOf(moment).isRequired,
+  }
+
+  constructor(props){
+    super(props)
+    getPrrrsPerWeek(props.week)
+    this.timeFromStartofWeekInSeconds = this.timeFromStartofWeekInSeconds.bind(this)
+    this.areaLeftofPoints = this.areaLeftofPoints.bind(this)
```

Comment:
you also dont need to bind this function

you only need to bind when you are giving a reference to the function away